app/src/main/res/values-es/strings.xml

```diff
@@ -162,9 +183,7 @@
     <!-- About Activity -->
     <string name="aboutActivityTitle">Acerca de nosotros</string>
     <string name="aboutDescription">
-        En DuckDuckGo, estamos sentando un precedente en la confianza en línea.\n\
-        El navegador privado de DuckDuckGo ofrece todas las funciones de privacidad esenciales que necesitas para protegerte mientras realizas búsquedas y navegas en Internet, lo que incluye el bloqueo de rastreadores, cifrado inteligente y búsqueda privada de DuckDuckGo.\n\
-        Después de todo, Internet no debería percibirse como una amenaza y disfrutar de la privacidad en línea que mereces debería ser lo más fácil posible.</string>
+        En DuckDuckGo, estamos sentando un precedente en la confianza en línea.\n\\ El navegador privado de DuckDuckGo ofrece todas las funciones de privacidad esenciales que necesitas para protegerte mientras realizas búsquedas y navegas en Internet, lo que incluye el bloqueo de rastreadores, cifrado inteligente y búsqueda privada de DuckDuckGo.\n\\ Después de todo, Internet no debería percibirse como una amenaza y disfrutar de la privacidad en línea que mereces debería ser lo más fácil posible.</string>
```

Comment:
I've seen this in different languages, not sure why that has changed but is breaking the about us screen, rather than adding a new line is adding an "/" symbol. They took `\n\` and made it `\n\\` escaping the `\` symbol. 

I take if we go back to the previous string everything will work again but the translation will get screwed up next time we get them. 

What about using <br/><br/> or trying with just `/n` or `\r\n` instead of `/n/`?

Reply:
I like the idea of replacing it with `<br></br>`